index.html

```diff
@@ -197,6 +196,27 @@ <h2>About Myself</h2>
         </ul>
       </div>
     </section>
+    <!-- Form -->
+    <section id="form" class="flex">
+      <form class="flex-column" action="https://formspree.io/f/{form_id}" method="post">
```

Comment:
- 🙏Please replace `{form_id}` with the actual ID from Formspree. If you log in to Formspree's service at [Formspree website](https://formspree.io/html), you will obtain the real ID. By doing so, your action will be submitted to the correct URL. Your code should resemble the following:
```suggestion
      <form class="flex-column" action="https://formspree.io/f/xoqzgjck" method="post">
```